airbyte-integrations/connectors/destination-aws-datalake/destination_aws_datalake/spec.json

```diff
@@ -6,20 +6,22 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "AWS Datalake Destination Spec",
     "type": "object",
-    "required": ["credentials", "region", "bucket_name", "bucket_prefix"],
+    "required": [
+      "aws_account_id",
+      "credentials",
+      "region",
+      "bucket_name",
+      "lakeformation_database_name",
+      "format"
```

Comment:
adding new required spec fields will make this a breaking change - any connection that already runs with an older version will break upon upgrade. Is it possible to adde defaults for `format`, `aws_account_id`,  and `lakeformation_database_name` to avoid making this a breaking change?

Reply:
I'll add defaults for `format` and `lakeformation_database_name`

Reply:
actually, it looks like `lakeformation_database_name` didn't have a default in the previous version and wasn't flagged as required. It most likely should be required, the connector needs to know about what database it should use by default?